lib/pages/scan/remote_button.dart

```diff
@@ -28,22 +40,45 @@ class RemoteButton extends StatelessWidget {
       return Container();
     }
 
-    return Padding(
-        padding: EdgeInsets.only(left: 8.0, top: 0.0, right: 8.0, bottom: 0.0),
-        child: TextButton(
-          style: ButtonStyle(
-            fixedSize: WidgetStateProperty.all<Size>(Size(double.infinity, 0)),
-            backgroundColor: WidgetStateProperty.all<Color>(Colors.red),
-            foregroundColor: WidgetStateProperty.all<Color>(Colors.white),
+    return Container(
+      width: 328,
+      height: 40,
+      decoration: BoxDecoration(
+        color: AppColors.defaultRed,
+        borderRadius: BorderRadius.circular(25),
+      ),
+      child: Row(
+        children: [
+          Expanded(
+            child: TextButton(
+              style: ButtonStyle(
+                foregroundColor: WidgetStateProperty.all<Color>(Colors.white),
+              ),
+              onPressed: () async {
+                PolaAnalytics.instance().donateOpened(code);
+                await launchUrl(
+                  url,
+                  mode: LaunchMode.externalApplication,
+                );
+              },
+              child: Text(buttonDto.title),
+            ),
+          ),
+          Container(
+            margin: const EdgeInsets.only(right: 8.0),
+            child: GestureDetector(
+              onTap: () {
+                setState(() {
+                  _isVisible = false;
```

Comment:
Po to robiliśmy zmiany w ScanBlocu, żeby użyć tu bloka i na podstawie ScanState decydować czy ten guzik ma być widoczny czy nie.